src/main/java/kostka/moviecatalog/entity/Comment.java

```diff
@@ -0,0 +1,50 @@
+package kostka.moviecatalog.entity;
+
+public class Comment {
```

Comment:
A jo, to máš tu třídu jak si se ptal, jestli můžeš použít z druhého balíčku, aby si to nemusel mít duplicitně?

Reply:
přesně tak, a taky jednu Exception :)